reference/conanfile/attributes/sources.inc

```diff
@@ -111,4 +111,25 @@ For example, a *conandata.yml* with information about sources that looks like th
 .. code-block:: python
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version])
\ No newline at end of file
+        tools.get(**self.conan_data["sources"][self.version])
```

Comment:
``tools.get`` no longer exist in Conan 2? This should have been fixed some time ago
```suggestion
        get(self, **self.conan_data["sources"][self.version])
```